SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialEventTracer.h

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Interop/Connection/SpatialSpanIdCache.h"
+#include "Interop/Connection/SpatialTraceEvent.h"
+
+// Documentation for event tracing in the GDK can be found here: https://brevi.link/gdk-event-tracing-documentation
+
+// TODO(EventTracer): make sure SpatialEventTracer doesn't break the LatencyTracer functionality for now (maybe have some macro/branching in
+// .cpp file, when the LatencyTracer is enabled?)
+// TODO(EventTracer): make sure the overhead of SpatialEventTracer is minimal when it's switched off
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialEventTracer, Log, All);
+
+// TODO - Individual RPC Calls (distinguishing between GDK and USER)
+// TODO - RPCs on newly created objects go through a different flow and need to be handled
+
+namespace worker
```

Comment:
The headers these things come from are tiny and don't include other things from the project, so they aren't really going to affect compile times in any noticeable way. I'd rather just include the functions than add namespace clutter here.